cmd/lifecycle/analyzer.go

```diff
@@ -225,3 +249,23 @@ func (a *analyzeCmd) restoresLayerMetadata() bool {
 func (a *analyzeCmd) platformAPIVersionGreaterThan06() bool {
 	return api.MustParse(a.platform.API()).Compare(api.MustParse("0.7")) >= 0
 }
+
+func (a *analyzeCmd) verifyReadAccess(name string) error {
```

Comment:
```suggestion
func (a *analyzeCmd) verifyReadAccess(imageRef string) error {
```

Reply:
Done in the following [commit](https://github.com/buildpacks/lifecycle/pull/646/commits/d264cb1197460b96220754740341e1a28520c062)